db/db.go

```diff
@@ -90,6 +90,9 @@ func getResourceIndex(index addrs.InstanceKey) string {
 
 func marshalAttributeValues(src *states.ResourceInstanceObjectSrc) (attrs []types.Attribute) {
 	vals := make(attributeValues)
+	if src == nil {
+		return 
+	}
```

Comment:
Could you open a separate PR for this please?

Reply:
Will do.